src/wlapplication_messages.cc

```diff
@@ -19,178 +19,273 @@
+	  {"", "replay", _("FILENAME"), _("Directly loads the replay FILENAME"), false},
+	  {"", "script", _("FILENAME"),
+		_("Run the given Lua script after initialization. Only valid with --scenario, --loadgame, or "
+		  "--editor"),
+		false},
+	  {"", "editor", "",
+		_("Directly starts the Widelands editor. You can add a =FILENAME to directly load the map "
+		  "FILENAME in editor"),
+		false},
+	  /// Misc
+	  {" ", "nosound", "", _("Starts the game with sound disabled"), false},
+	  {" ", "fail-on-lua-error", "", _("Force Widelands to crash when a Lua error occurs"), false},
+	  {"", "ai_training", "",
+		_("Enables AI training mode. See https://www.widelands.org/wiki/Ai%20Training/ for a full "
+		  "description of the AI training logic"),
+		false},
+	  {"", "auto_speed", "",
+		_("In multiplayer games only, this will keep adjusting the game speed "
+		  "automatically, depending on FPS. Useful in conjunction with --ai_training"),
+		false},
+	  /// Saving options
+	  {"", "autosave",
+		/** TRANSLATORS: A placeholder for a numerical value */
+		_("n"),
+		/** TRANSLATORS: `n` references a numerical placeholder */
+		_("Automatically save each `n` minutes"), false},
+	  {"", "rolling_autosave", _("n"),
+		/** TRANSLATORS: `n` references a numerical placeholder */
+		_("Use `n` files for rolling autosaves"), true},
+	  {"", "nozip", "", _("Do not save files as binary zip archives"), false},
+	  {"", "display_replay_filenames", _("[true*|false]"), _("Show filenames in replay screen"),
+		true},
+	  {"", "editor_player_menu_warn_too_many_players", _("[true*|false]"),
+		_("Enable verbose debug messages"), true},
+	  /// Game options
+	  {"", "auto_roadbuild_mode", _("[true*|false]"), _("Start building road after placing a flag"),
+		true},
+	  {"", "display_flags", _("[...]"), _("Display flags to set for new games"), true},
+#if 0  // TODO(matthiakl): Re-add training wheels code after v1.0
+	{"",
+	 "training_wheels",
+	 _("[true*|false]"),
+	 "",
+	 true
+	},
+#endif
+	  {"", "edge_scrolling",
+		/** TRANSLATORS: You may translate true/false, also as on/off or yes/no, but */
+		/** TRANSLATORS: it HAS TO BE CONSISTENT with the translation in the widelands textdomain. */
+		/** TRANSLATORS: * marks the default value */
+		_("[true|false*]"), _("Scroll when the mouse cursor is near the screen edge"), true},
+	  {"", "numpad_diagonalscrolling", _("[true|false*]"),
+		_("Allow diagonal scrolling with the numeric keypad"), true},
+	  {"", "game_clock", _("[true|false*]"), _("Display system time in the info panel"), true},
+	  {"", "ctrl_zoom", _("[true|false*]"), _("Zoom only when Ctrl is pressed"), true},
+	  {"", "single_watchwin", _("[true|false*]"), _("Use single watchwindow mode"), true},
+	  {"", "transparent_chat", _("[true*|false]"),
+		_("Show in-game chat with transparent background"), true},
+	  {"", "toolbar_pos", _("[...]"), _("Sets the toolbar location and mode"), true},
```

Comment:
Add a translators note that this does not use the `…` character on purpose.

Reply:
Is this true? It could also be `…`, if someone wants it…

Reply:
With a monospaced font as most terminals use, `…` is harder to see than `...`, so the latter should be preferred in this case.

Reply:
What are supported modes? are they translated, too?